TAP version 13
ok 1 - this is fine
not ok 2 - Error: catch!
  ---
  {"at":{"column":7,"file":"test/test/throw-root.js","function":"Object.<anonymous>","line":3},"message":"Error: catch!","source":"throw new Error('catch!')\n","test":"TAP"}
  ...

Bail out! # Error: catch!

